components/core/src/Theme.js

```diff
@@ -22,41 +20,32 @@ export const useTheme = () => {
 	return themeObject;
 };
 
-export const useIsKeyboardUser = () => {
-	const [isKeyboardUser, setIsKeyboardUser] = useState(false);
+// ==============================
+// Component
+// ==============================
 
-	// Handle keys
+export const GEL = ({ brand, children, ...props }) => {
 	const keyHandler = event => {
 		if (event.key === 'Tab') {
-			setIsKeyboardUser(true);
+			document.getElementsByTagName('body')[0].classList.remove('isMouseMode');
```

Comment:
We need to work on this. This needs to be a `<javascript>` tag we add to the dom to make sure we can check that it already happened... Will help with this!

Reply:
Sounds good. Ill do it in a separate pr.